wisetools.py

```diff
@@ -127,7 +127,7 @@ def flush(readBuff, counts):
 		if stairSize <= threshold or threshold < 0:
 			for read in readBuff:
 				location = read.pos / binsize
-				counts[location] += 1
+				counts[int(location)] += 1
```

Comment:
I'm fine with this change but I'd like to know, is this a python version specific behaviour? `binsize` and `read.pos` should be integers (per definition), so how did this produce a 'float index calling warning'?